src/components/MarketGraph/index.js

```diff
@@ -114,6 +104,90 @@ const MarketGraph = ({ data = [], market: { event: { type, lowerBound, upperBoun
   return <div />
 }
 
+const CustomTooltip = ({ payload = [], label, active, separator, itemStyle, itemSorter, labelStyle, wrapperStyle }) => {
```

Comment:
Please define this in a seperate file e.g. src/components/CustomTooltip/index.js